getting-started/installation/adding-private-nuget-feed.md

```diff
@@ -9,19 +9,13 @@ published: True
 position: 5
 ---
 
-# Adding the Telerik Private NuGet Feed to VS
+# Adding the Telerik NuGet Feed to Your System
```

Comment:
I think it's more correct to say 'your project or solution' than 'your system'

Reply:
Both approaches (Package Manager and CLI) let you add the NuGet source system-wide. It will be available for any project you develop.